identity/test/pool.go

```diff
@@ -759,7 +756,7 @@ func TestPool(ctx context.Context, conf *config.Config, p persistence.Persister,
 
 			t.Run("not if on another network", func(t *testing.T) {
 				_, on := testhelpers.NewNetwork(t, ctx, p)
-				actual, err := on.ListIdentities(ctx, identity.ListIdentityParameters{
+				actual, _, err := on.ListIdentities(ctx, identity.ListIdentityParameters{
```

Comment:
Do we have any test that asserts that the `nextPage` returned here is the correct page?

Reply:
added an assertion